### DISCUSSION TITLE
{Application} Filet Finance - Decentralized Filecoin Staking Protocol

### DISCUSSION THREAD

**Discussion by xtop-protocol** [filecoin-project/community#468] [Project Showcase]
Filet is a Filecoin mining power tokenization protocol that runs on the blockchain. It tokenizes Filecoin mining power and introduces it into the DeFi ecosystem to provide FIL holders with high-growth FIL staking services, and the annualized FIL return is as high as 20% APY. 

Filet is a Dapp deployed on the Filecoin (FVM), BSC and HECO, and is connected to the Filecoin chain. The mining power and assets are completely open and transparent.  Users' assets are safe and free to trade, with high returns. 

Website
**www.filet.finance**

Audit Report
Filet: https://www.filet.finance/REP-Filet%20Staking-2021-06-22.pdf

Whitepaper
https://www.filet.finance/FILET%20Whitepaper_EN_V1.1.0.pdf

Website: www.filet.finance
Telegram: t.me/filet_finance
Twitter: @Filet_finance
Medium: https://medium.com/@filet.finance
Email: contact@filet.finance
WeChat: filet520

**Comment by BunnyJ0209** [comment]
It's now over 100,000 FIL staked.

**Reply by xtop-protocol** [reply]
FYI. The accumulated stakings is over 160K FIL.

**Reply by lherfel** [reply]
so is filet safe then?

https://native.filet.finance/

Is there some sort of Telegram or discord channel for filecoin?

Thanks

**Reply by xtop-protocol** [reply]
Here is the official TG link: https://t.me/filet_finance

For the link you mentioned, it is the one for native FIL staking on Filet, you can also visit Filet via www.filet.finance.

**Comment by xtop-protocol** [comment]
Over 330,000 FIL staked in Filet, and it is still growing. A good platform to stake FIL to get more FIL from the real Filecoin mining.

**Comment by xtop-protocol** [comment]
Filet will join Filecoin Countdown to FVM event on Mar. 1st. There will be a showcase about what Filet Finance does and how it works with FVM to serve the Filecoin ecosystem. 

We will also prepare some small surprise for new friends. See you in Denver! 😉
Check more: 
https://events.fil.org/fvm-hackerbase/page/2570587/countdown-to-fvm-

**Comment by xtop-protocol** [comment]
Just an update here, Filet was officially launched on FVM on May 9th, 2023!
